src/app/views/discover/games/view/devlog/view/view.ts

```diff
@@ -69,6 +69,8 @@ export default class RouteDiscoverGamesViewDevlogView extends BaseRouteComponent
 
 	post: FiresidePost | null = null;
 
+	permalinkWatcher?: Function;
```

Comment:
I think we should rename these to make it clearer this IS the deregister. the current name is misleading.